main.go

```diff
@@ -15,12 +15,12 @@
 package main
 
 import (
-	"github.com/fairwindsops/polaris/cmd/polaris"
+	cmd "github.com/fairwindsops/polaris/cmd/polaris"
```

Comment:
My editor kept wanting to change this so even though the module at that directory is named `cmd` this makes the below code more clear I think.

Reply:
👍